sdk/appconfiguration/app-configuration/README.md

```diff
@@ -80,12 +80,12 @@ const client = new AppConfigurationClient("<connection string>");
 
 ## Key concepts
 
-The [`AppConfigurationClient`](https://docs.microsoft.com/javascript/api/@azure/app-configuration/appconfigurationclient) has some terminology changes from App Configuration in the portal.
+The [`AppConfigurationClient`](https://aka.ms/azsdk/js/docs/ref/app-configuration/appconfigurationclient) has some terminology changes from App Configuration in the portal.
```

Comment:
Hmm, I can see doing this for type/method docs would require us to create and maintain many aka.ms links.  I feel it would be fine to only use aka.ms links for landing pages but would like to know what others think.

Reply:
Yea, good question. @jongio I think you wanted to keep these links just for general docs, this many layers deep seems to be a bit much and a pain to maintain

Reply:
I think we should do one for each landing page.

Reply:
Do you mind providing some guidance on the naming of the links for these landing pages? The one referenced in this thread happened automatically.

Reply:
I confirmed with Jeremy and I will proceed to focus on links like:

```
[API reference documentation](https://aka.ms/azsdk/js/docs/ref/app-configuration)
```

Not like:

```
[`AppConfigurationClient`](https://aka.ms/azsdk/js/docs/ref/app-configuration/appconfigurationclient)
```

Reply:
So this will be for general ref docs but not for that particular client, right? Will you keep the links for client the way they were before?

Reply:
Ok, latest directions: Jon says that `docs.microsoft.com don't need aka` . I'll only do aka links to non-docs.microsoft links.

Reply:
@kaerm Hi! I will be making a new PR and closing this one. I believe this list gets closer to those links that should be moved to aka.ms: https://gist.github.com/sadasant/7f4f00f433577b66fd016136c4312e7e Do you mind helping me know if this list is correct? or if I could narrow it even further? I will do the PR from your feedback, for sure.

Reply:
So you are planning to aka.ms the links in that list, correct?

Reply:
@kaerm yep! I'm doing other things, but I will get back to this asap.